gui/services/migrations/0195_auto__add_field_nfs_nfs_srv_mountd_log__add_field_nfs_nfs_srv_statd_lo.py

```diff
@@ -0,0 +1,435 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Adding field 'NFS.nfs_srv_mountd_log'
+        db.add_column(u'services_nfs', 'nfs_srv_mountd_log',
+                      self.gf('django.db.models.fields.BooleanField')(default=True),
+                      keep_default=False)
+
+        # Adding field 'NFS.nfs_srv_statd_lockd_log'
+        db.add_column(u'services_nfs', 'nfs_srv_statd_lockd_log',
+                      self.gf('django.db.models.fields.BooleanField')(default=False),
```

Comment:
I have checked the table in sqlite3. It is getting saved in the form of 0/1 only.
But in line 14, I'll change the value to keep_default=True.
As Josh wanted it to be True by default.